h3/src/error.rs

```diff
@@ -232,7 +254,6 @@ impl Error {
         matches!(&self.inner.kind, Kind::HeaderTooBig { .. })
     }
 
-    #[cfg(feature = "test_helpers")]
```

Comment:
It'd be good to keep this as an "unstable" method. The `Kind` is not meant to be matched on outside of `h3` (and the tests), since we might change it at anytime.